### DISCUSSION TITLE
Is there a plan to add support for oracle database?

### DISCUSSION THREAD

**Discussion by zhaopinglu** [rbatis/rbatis#293] [General]
Is there a plan to add support for oracle database?

**Comment by zhuxiujia** [comment]
> Is there a plan to add support for oracle database?

There are no plans for this, currently SQL drivers use ``` sqlx-core```

If there were a better SQL driver（maybe）

Thank you

**Comment by zhuxiujia** [comment]
see https://crates.io/crates/rbdc-oracle
